sdk/spring-sdk/src/it/http/test-requests.http

```diff
@@ -0,0 +1,14 @@
+POST {{url}}/user/{{userId}}/email/andrzej
+Content-Type: application/json
```

Comment:
I can remove it, but it's pretty convenient to have some predefined requests directly in IDE.

Reply:
You can also use a readme with curl commands inside `shell` code blocks and it's similarly convenient (you need to repeat host and port I guess). We have been using that I think, [example](https://github.com/lightbend/kalix-jvm-sdk/blob/main/samples/java-valueentity-shopping-cart/README.md).

Reply:
Interesting, I wasn't aware of that, thanks!